AzureFunctions.EventGrid/EventGridAsyncCollector.cs

```diff
@@ -1,29 +1,33 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Net.Http;
+using System.Text;
 using System.Threading;
 using System.Threading.Tasks;
-using Microsoft.Azure.EventGrid;
-using Microsoft.Azure.EventGrid.Models;
 using Microsoft.Azure.WebJobs;
+using Newtonsoft.Json;
 
 namespace AzureFunctions.EventGrid
 {
     public class EventGridAsyncCollector : IAsyncCollector<Event>
     {
-        private readonly string topicHostname;
-        
+        private readonly string topicEndPoint;       
+        private readonly string topicKey; 
+
         private readonly IList<Event> eventCollection;
-        private readonly TopicCredentials topicCredentials;
+        private readonly EventGridAttribute attribute;
+        
+        private HttpClient httpClient;
```

Comment:
If you make this one `static`, it can be reused across multiple invocations of the client Function App.

Reply:
fixed with `static`